src/MapViewController.swift

```diff
@@ -17,7 +17,21 @@ public class MapViewController: TGMapViewController, LocationManagerDelegate, TG
     var lastSetPoint: TGGeoPoint?
     var shouldShowCurrentLocation = false
     public var shouldFollowCurrentLocation = false
-    public var findMeButton = MapViewController.createFindMeButton()
+    public var findMeButton = UIButton(type: .Custom)
```

Comment:
Does `findMeButton` need to be initialized here when its going to be set by `createFindMeButton()` in the constructor?